stdlib/3/lib2to3/pgen2/driver.pyi

```diff
@@ -0,0 +1,24 @@
+# Stubs for lib2to3.pgen2.driver (Python 3.6)
+
+from typing import Any, Callable, IO, Iterable, List, Optional, Tuple
+
+from logging import Logger
+from lib2to3.pytree import _Context, _NL
+from lib2to3.pgen2.grammar import Grammar
+
+_RawNode = Tuple[int, str, _Context, List[_NL]]
```

Comment:
The last element should be Optional I think. See the implementation of Parser.shift.